src/framework/linalg/enable_if_numeric.hpp

```diff
@@ -19,20 +19,27 @@
 #include <type_traits>
 
 // Type check template to enable functions if type is a numeric scalar
-// (integer, float, or complex float)
```

Comment:
This was specifically to check all all numerics including complex.

If you wanted one that doesn't include complex I would make that the special case below with
`enable_if_numeric_t`  # real or complex
`enable_if_real_numeric_t` # real only
`enable_if_complex_numeric_t` # complex only

Reply:
oh! sure.. I just forgot to add the `complex floating-points` part. I just wanted to make sure that `double` types are included (and not just  `float`)